homeassistant/bootstrap.py

```diff
@@ -412,16 +411,23 @@ def _ensure_loader_prepared(hass: core.HomeAssistant) -> None:
         loader.prepare(hass)
 
 
+def _persistent_notification(hass: core.HomeAssistant, component: str,
+                             link: Optional[bool]=False):
+    """Print a persistent notification."""
+    _PERSISTENT_ERRORS[component] = _PERSISTENT_ERRORS.get(component) or link
+    _lst = [HA_COMPONENT_URL.format(name) if link else name
+            for name, link in _PERSISTENT_ERRORS.items()]
+    message = ('The following components and platforms could not be set up:\n'
```

Comment:
Another tip: add a link at the end to `/dev-info` so they can see the validation errors.

```markdown
[See validation errors](/dev-info)
```